virttest/utils_disk.py

```diff
@@ -1046,6 +1046,24 @@ def get_disk_by_serial(serial_str, session=None):
             return disk
 
 
+def dd_data_to_vm_disk(vm, disk, bs='1M', seek='0', count='100'):
```

Comment:
maybe use session as parameter rather than vm since we have two types of VM object:qemu-vm and libvirt-vm

Reply:
updated